### ISSUE TITLE
reproduce problems

### ISSUE THREAD

**Issue by panjunhao99** [Dootmaan/MT-UNet#21]
I tried to reproduce your code on acdc, and found that the validation set always performed better in the early stage (around 50 epoch) during the training process, and always performed worse in the later epoch, which made me not sure which model to choose. How can I improve to find the most accurate model faster?

**Comment by Dootmaan** [comment]
Hi @panjunhao99 and thank you for your question.

Usually we will choose the model with the best validation score for testing but meanwhile we can also test the models around it. The best test score if often obtained after several rounds of training and model selecting. If you find the validation score falls quickly in training, maybe you could try to lower the learning rate, or simply kill the process and load the newest weights to start another round of training.

**Comment by Dootmaan** [comment]
*This issue is closed since no further activity has happened for a while.*
